index.html

```diff
@@ -1469,6 +1469,36 @@
   </div>
   <!-- ends QCTools Report (no audio) -->
 
+  <!-- Read/Extract EIA-608 Closed Captions -->    
+  <span data-toggle="modal" data-target="#readeia608"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Read or extract EIA-608 (Line 21) closed captioning">Read/Extract EIA-608 Closed Captioning</button></span>
+  <div id="readeia608" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog modal-lg">  
+      <div class="modal-content">
+        <div class="well">
+          <h3>Read/Extract EIA-608 (Line 21) closed captioning</h3>
+          <p><code>ffprobe -f lavfi -i movie=<i>input_file</i>,readeia608 -show_entries frame=pkt_pts_time:frame_tags=lavfi.readeia608.0.cc,lavfi.readeia608.1.cc -of csv > <i>input_file</i>.csv</code></p>
+          <p>This command uses FFmpeg's <a href="https://ffmpeg.org/ffmpeg-filters.html#readeia608" target="_blank">readeia608</a> filter to extract the hexidecimal values hidden within <a href="https://en.wikipedia.org/wiki/EIA-608" target="_blank">EIA-608 (Line 21)</a> Closed Captioning, outputting a csv file. For more information about EIA-608, check out Adobe's <a href="https://www.adobe.com/content/dam/Adobe/en/devnet/video/pdfs/introduction_to_closed_captions.pdf" target="_blank">Introduction to Closed Captions</a>.</p>
+          <p>If hex isn't your thing, closed captioning <a href="http://www.theneitherworld.com/mcpoodle/SCC_TOOLS/DOCS/CC_CHARS.HTML" target="_blank">character</a> and <a href="http://www.theneitherworld.com/mcpoodle/SCC_TOOLS/DOCS/CC_CODES.HTML" target="_blank">code</a> sets can be found in the documentation for SCTools.</p>   
+          <dl>
+            <dt>ffprobe</dt><dd>starts the command</dd>
+            <dt>-f lavfi</dt><dd>tells ffprobe to use the Libavfilter input virtual device <a href="http://ffmpeg.org/ffmpeg-devices.html#lavfi" target="_blank">[more]</a></dd>
+            <dt>-i</dt><dd>input file and parameters</dd>
```

Comment:
Minor, but ffmprovisr convention is to include _input_file_ after the input flag, like so:
`<dt>-i <i>input_file</i></dt>` (etc)